source/technical/daypass.rst

```diff
@@ -0,0 +1,91 @@
+.. _daypass:
+
+=======
+Daypass
+=======
+Normally, only Chameleon users with active allocations are able to launch and
+view Trovi artifacts. To allow anyone to launch an artifact, we also provide
+daypass. This allows for a non-Chameleon user to have access to Chameleon
+for a limited amount of time, using a small, separate allocation.
+People interested in reproducing your project will send requests
+to the managers of a project. If approved, the requesting user will recieve an
+email invitation to join a reproducibility project. When they accept, they
```

Comment:
```suggestion
email invitation to join your reproducibility project. When they accept, they
```
?

Reply:
I changed this so users do not think they are joining the user's main project (e.g. it's a reproducibility project). Perhaps a better wording is "a separate reproducibility project".